packages/frontend/src/pages/faq/FaqPage.tsx

```diff
@@ -11,14 +11,117 @@ export interface FaqPageProps {
 }
 
 export function FaqPage(props: FaqPageProps) {
+  const linkClasses =
+    'mt-4 first:mt-0 text-gray-850 hover:text-pink-900 dark:text-white/80 dark:hover:text-pink-200 font-semibold text-base'
   return (
     <>
       <Navbar {...props.navbar} />
-      <PageContent narrow>
-        <article
-          className="Faq"
-          dangerouslySetInnerHTML={{ __html: props.content }}
-        />
+      <PageContent>
+        <span className="mt-16 flex flex-col border-b border-gray-850 pb-12">
```

Comment:
The FAQ should be autogenerated from the markdown. This can be achieved using a toc plugin for our markdown renderer I think

Reply:
Are you saying we need to implement the content table within MD? I don't believe it's possible

Reply:
We are using the `markdown-it` library. It supports plugins. We could use for example https://github.com/cmaas/markdown-it-table-of-contents and have the toc be created for us automatically.